geonode/api/api.py

```diff
@@ -87,6 +87,42 @@ class Meta:
         }
 
 
+class RegionResource(TypeFilteredResource):
```

Comment:
I would avoid the code duplication in this class and TagResource. At least the dehydrate_count method should be in common.